launcher/container_runner.go

```diff
@@ -389,8 +426,17 @@ func (r *ContainerRunner) fetchAndWriteToken(ctx context.Context) error {
 				r.logger.Printf("token refreshing stopped: %v", ctx.Err())
 				return
 			case <-timer.C:
-				// Refresh token.
-				duration, err := r.refreshToken(ctx)
+				var duration time.Duration
+				// Refresh token with default retry policy.
+				err := backoff.RetryNotify(
```

Comment:
So if retry backoff go over the MaxElapsedTime or retried about 5 times, it will never retry again? (I thought it will retry every 1 hr indefinitely, but I think this is also okay)

Reply:
Correct. We can always add indefinite retries. Either way, there will be some time period with hourly intervals with an expired token. So, workloads will have to rearchitect against this anyway. We can always change this if we find that users want this functionality, but this is simpler for now.